serf/serf.go

```diff
@@ -751,6 +745,13 @@ func (s *Serf) Members() []Member {
 	return members
 }
 
+func (s *Serf) membersLength() int {
```

Comment:
Hi @tyler I knew this seemed familiar :-) There's an existing `NumNodes()` method on the `Serf` you can call that does this.